cmake/SlangTarget.cmake

```diff
@@ -245,6 +245,13 @@ function(slang_add_target dir type)
         )
     endforeach()
 
+    # set public include folder for all target
+    set(publicInc_abs "${PROJECT_SOURCE_DIR}/include")
+    target_include_directories(
+        ${target}
+        PUBLIC "$<BUILD_INTERFACE:${publicInc_abs}>"
```

Comment:
This is definitely not a good thing to do as `PROJECT_SOURCE_DIR` could refer to a higher level project that includes slang. Please use `slang_SOURCE_DIR` for this sort of thing.

However in this specific case, I don't think adding it to every target is a good thing (see the broader review comment)